ui/views/mus/desktop_window_tree_host_mus.cc

```diff
@@ -487,8 +490,13 @@ aura::WindowTreeHost* DesktopWindowTreeHostMus::AsWindowTreeHost() {
 }
 
 void DesktopWindowTreeHostMus::ShowWindowWithState(ui::WindowShowState state) {
+#if defined(OS_LINUX) && defined(USE_OZONE) && !defined(OS_CHROMEOS)
+  // On Ozone/Linux we need to notify the server of any show-state change.
+  window()->SetProperty(aura::client::kShowStateKey, state);
+#else
   if (state == ui::SHOW_STATE_MAXIMIZED || state == ui::SHOW_STATE_FULLSCREEN)
     window()->SetProperty(aura::client::kShowStateKey, state);
```

Comment:
What would happen if the show property state is changed for chrome os case as well? 
What I want to say is whether it's possible to unify these lines and just have window()->SetProperty() for both OS_LINUX and OS_CHROMEOS.

Reply:
I think we can try, sir. Lets see what sky@ says.